### ISSUE TITLE
[bug] Binary Switch - Hass Discovery issue after start/restart

### ISSUE THREAD

**Issue by billiaz** [zwave-js/zwavejs2mqtt#50]
**Version**

Build/Run method
- [x] Docker
- [ ] PKG
- [x] Manually built (git clone - npm install - npm run build ) 

zwavejs2mqtt version: latest Master

When you .first start the application. A first discovery is generated by the app. On Binary switches this looks like:
```
{
  "type": "switch",
  "object_id": "switch",
  "discovery_payload": {
    "payload_off": false,
    "payload_on": true,
    "value_template": "{{ value_json.value }}",
    "command_topic": "zwave2mqtt/Guestroom/switch/37/1/currentValue/set",
    "state_topic": "zwave2mqtt/Guestroom/switch/37/1/currentValue",
    "device": {
      "identifiers": [
        "zwavejs2mqtt_0xcddb486e_node66"
      ],
      "manufacturer": "Fibargroup",
      "model": "Single Switch 2 (FGS213)",
      "name": "Guestroom-switch",
      "sw_version": "3.3"
    },
    "name": "Guestroom-switch_switch",
    "unique_id": "zwavejs2mqtt_0xcddb486e_66-37-1-currentValue"
  },
  "discoveryTopic": "switch/Guestroom-switch/switch/config",
  "values": [
    "37-1-currentValue",
    "37-1-targetValue"
  ],
  "persistent": false,
  "ignoreDiscovery": false,
  "id": "switch_switch"
}
```
As we can see commandTopic is wrong, as currentValue is read only, and does not accept set commands
Once we click rediscover node we get the right payload:
```
{
  "type": "switch",
  "object_id": "switch",
  "discovery_payload": {
    "payload_off": false,
    "payload_on": true,
    "value_template": "{{ value_json.value }}",
    "command_topic": "zwave2mqtt/Guestroom/switch/37/1/targetValue/set",
    "state_topic": "zwave2mqtt/Guestroom/switch/37/1/currentValue",
    "device": {
      "identifiers": [
        "zwavejs2mqtt_0xcddb486e_node66"
      ],
      "manufacturer": "Fibargroup",
      "model": "Single Switch 2 (FGS213)",
      "name": "Guestroom-switch",
      "sw_version": "3.3"
    },
    "name": "Guestroom-switch_switch",
    "unique_id": "zwavejs2mqtt_0xcddb486e_66-37-1-currentValue"
  },
  "discoveryTopic": "switch/Guestroom-switch/switch/config",
  "values": [
    "37-1-currentValue",
    "37-1-targetValue"
  ],
  "persistent": false,
  "ignoreDiscovery": false,
  "id": "switch_switch"
}
```

I tried to add at lib/Gateway.js line 1320 the following:
```
cfg.discovery_payload.command_topic = setTopic
```
Which converts the command_topic on first startup/restart to null.
The strange part is values contain Target Value. I tried to find where in code this can be causing it. But still I have no clue.

**Comment by robertsLando** [comment]
@billiaz I think it may be related to #38

**Comment by billiaz** [comment]
pull request to fix the issue
https://github.com/zwave-js/zwavejs2mqtt/pull/52

**Comment by robertsLando** [comment]
Fixed by #54
